packages/eslint-config/eslintrc.js

```diff
@@ -243,6 +243,22 @@ module.exports = {
         selector: 'typeLike',
         format: ['PascalCase'],
       },
+
+      // Ignore properties that require quotes
+      {
+        selector: [
+          // 'classProperty',
+          'objectLiteralProperty',
+          // 'typeProperty',
+          // 'classMethod',
+          // 'objectLiteralMethod',
+          // 'typeMethod',
+          // 'accessor',
+          // 'enumMember',
+        ],
+        format: null,
+        modifiers: ['requiresQuotes'],
+      },
```

Comment:
https://github.com/loopbackio/loopback-next/blob/3d6dd2ef3ae55a65f907ee2da99ca57d285fd061/packages/eslint-config/eslintrc.js already tried to solve this issue. But it seems like it broke. Can we merge this with that?

Reply:
There are many cases to complete... A list would be:
- Dotted (127.0.0.1)
- With hyphens (x-visibility)
- With slashes (application/x-www-form-urlencoded)
- With HTTP Codes (2xx, 3xx, 4xx & 5xx)
- With square brackets (filter[where][id])
- With curly braces ({id})
- With numbers (999)